content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
+local tax laws.
+
+To view all the available reports for the database, including all the localization-specific ones,
+navigate to :menuselection:`Payroll app --> Reporting` to view the available reports. Click on a
+specific report to view it.
 
 .. image:: reporting/reports.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Report dashboard view showing extra reports for Belgium databases.
 
-If a report is unavailable to a company, a user error pops up, stating `You must be logged in to a
-Belgian company to use this feature.`
+If a report is unavailable to a user, an :guilabel:`Invalid Operation` pop-up window appears,
+stating: `You must be logged in to a (country) company to use this feature.`
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+Filters
+=======
 
-.. image:: reporting/custom-filter.png
-   :align: center
-   :alt: Report dashboard view.
+At the top of each report, the default filters are shown inside the :guilabel:`Search` box.
 
-Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
-:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
-spreadsheet`.
+Click the :guilabel:`drop-down arrow` in the search box to display the available
+:guilabel:`filters`. Filters show information that matches the specific filter parameters.
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+.. example::
+   The :guilabel:`Work Entries Analysis` report has two default filters, the :guilabel:`Current
+   month:(Month)(Year)` filter, and the :guilabel:`Validated` filter. The :guilabel:`Payroll` report
+   only has one default filter, the :guilabel:`Last 365 Days Payslip` filter.
+
+   .. image:: reporting/custom-filter.png
+      :align: center
+      :alt: Filters enabled for the Work Entries Analysis report.
+
+All reports can include custom filters or group information by different metrics (employee,
+department, company, etc.). Some reports have the option to compare the current report to the
+previous time period or year (a :guilabel:`Comparison` option).
+
+Click on a parameter to select it and activate it. The report is immediately updated with the new
+parameters.
+
+The updated report can be set as a favorite report, meaning the parameters are stored for quick
+access in the future. First, click :guilabel:`Save the current search` under the
+:guilabel:`Favorites` section, and two options and a :guilabel:`Save` button appear.
```

Comment:
```suggestion
:guilabel:`Favorites` section, located in the search bar drop-down mega menu of filter options. Doing so reveals two options and a :guilabel:`Save` button appears.
```
please be as specific as possible when explaining steps like this. it helps avoid any confusion with readers. thanks 👍